packages/svelte/src/compiler/phases/3-transform/client/types.d.ts

```diff
@@ -60,6 +60,7 @@ export interface ComponentClientTransformState extends ClientTransformState {
 	};
 	readonly preserve_whitespace: boolean;
 
+	readonly optimisation_oneling: boolean;
```

Comment:
This could use a comment. I 'm not sure I'd know what `optimisation_oneling` meant from the name alone